dnf.spec

```diff
@@ -174,6 +174,9 @@ mv $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-2 $RPM_BUILD_ROOT%{_bindir}/dnf-autom
 rm $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-3
 %endif
 
+# This will eventually be the new default location for repo files
+mkdir $RPM_BUILD_ROOT%{_sysconfdir}/distro.repos.d
```

Comment:
The `mkdir` call still has to be done in `%install` or otherwise `%ghost` doesn't work. RPM won't actually install the directory, but it needs to exist.

cf.: http://www.rpm.org/max-rpm-snapshot/s1-rpm-inside-files-list-directives.html